src/libs/mps_comm/spdlog.mk

```
@@ -0,0 +1,23 @@
+#*****************************************************************************
+#                      Makefile Build System for Fawkes
+#                            -------------------
+#   Created on Mon 04 May 2020 10:30:53 CEST
+#   Copyright (C) 2020 by Till Hofmann <hofmann@kbsg.rwth-aachen.de>
+#
+#*****************************************************************************
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#
+#*****************************************************************************
+
+ifneq ($(PKGCONFIG),)
+  HAVE_SPDLOG= $(if $(shell $(PKGCONFIG) --exists spdlog; echo $${?/1/}),1,0)
```

Shouldn't this be:
```suggestion
  HAVE_SPDLOG= $(if $(shell $(PKGCONFIG) --exists spdlog; echo $${?/0/}),1,0)
```
Also, why is the if needed in the first place? can't we do: 
```suggestion
  HAVE_SPDLOG=$(shell $(PKGCONFIG) --exists spdlog; echo $$?)
```